### ISSUE TITLE
[Form Validation] Bug with validating array of fields

### ISSUE THREAD

**Issue by fpayer** [Semantic-Org/Semantic-UI#2688]
From the JSFiddle posted in #2683, if the first field in the array is valid, the entire form is marked valid regardless of the other fields.

http://jsfiddle.net/92p8fkbb/

Entering the the sequence: 1, a, a will have the form marked as valid.

**Comment by jlukic** [comment]
The current form supports `1` identifier 1 set of rules. 1 to many is not supported.

**Comment by federicotarantino** [comment]
A simil bug is for multiple dropdown.
For example, if I add an "email" validation on multiple dropdown (with allowAdditions), check is not for each email, but for all email as single string.

**Comment by shivekundefined** [comment]
Is this fixed in latest version of Semantic UI?

**Comment by fpayer** [comment]
Bug still exists

**Comment by ManfrediPist** [comment]
Yep, the bug hasn't beed fixed yet. Could anyone give us some new about it? It's really frustrating because I can't make any validation on each row of inputs I clone for my form 

XInputValue   YInputValue.  -> (Add One more row button)
Results in
XInputValue   YInputValue
XInputValue   YInputValue

But the validation only works on the first row on XInputValue   YInputValue

**Comment by goonursultan** [comment]
Зайбал жауап берсендерш щщщс

**Comment by lubber-de** [comment]
Individual field error handling for input array formfields is now fixed in https://github.com/fomantic/Fomantic-UI/pull/387
See http://jsfiddle.net/a1cnph4f/1/
Supposed to be part of upcoming Fomantic-UI 2.7,2

**Comment by snowshine09** [comment]
It seems the shorthand version could not be used in conjunction with the full version for validation rules. It would be more reasonable to be flexible in these forms.

**Comment by lubber-de** [comment]
@snowshine09 Support of mixing short and full field rules has been supported/fixed in the community fork [Fomantic-UI](https://fomantic-ui.com) by https://github.com/fomantic/Fomantic-UI/pull/1810
